### ISSUE TITLE
Allow hiding graph

### ISSUE THREAD

**Issue by danpowell88** [Rytiggy/api-watchface#5]
It would be good to have a mode where the number is shown large of the screen and perhaps you can swipe across to the graph (not sure if swiping is supported).

**Comment by Rytiggy** [comment]
These are great! I will incorporate something in to the app

**Comment by PedanticAvenger** [comment]
Along with the numerical display, perhaps an arrow that indicates trend similar to what we see in the xDrip UI? Still a swipe to the graph but it's a small element that provides a lot of context.

**Comment by PedanticAvenger** [comment]
Alright, I'm a poor gimp user, and I've blatantly ripped off features from other watch faces here but should be a reasonable discussion start point.  Tap/swipe graph on the left and a proposal to start bashing for the main screen on the right.  I really miss some of the clearer activity tracker data on this currently and like what the face "Flashring" has with regards to info and presentation.  Arrows under the BG value and fill the left with clock and day/date? I guess I also should have included units under or next to the last polled value on the main screen

![discussion](https://user-images.githubusercontent.com/15637097/34971468-f26f5550-fa37-11e7-92ba-5ff3d08df3e0.png)

**Comment by PedanticAvenger** [comment]
Commenting to record location of example to handle multiple screens.

https://community.fitbit.com/t5/SDK-Development/Wanted-example-of-multiple-UI-views-and-back-navigation/m-p/2300847

https://community.fitbit.com/t5/SDK-Development/Multi-screen-App-Template-Ionic-Views-v1-1/m-p/2414162#M2258

**Comment by PedanticAvenger** [comment]
![xdrip flashring-merge-screenshot](https://user-images.githubusercontent.com/15637097/35174915-64d8aa30-fd2e-11e7-94d1-b0e59797b931.png)
Thoughts?  I found out that the flashring watchface code is available OSS and started tweaking it for a front screen.  Data on the upper right is all static currently, none of @Rytiggy companion or display code integrated yet.  time below mmol/L is a "time since last update" placeholder.

**Comment by Rytiggy** [comment]
I love this! We could make it  so in the companion app there's a watchface select section!

**Comment by PedanticAvenger** [comment]
This one certainly needs something to bring up the graph.  Considering making the BG display space a "button" if I can do that as a clear overlay.  I'm going to put this on github shortly so it can be perused.

**Comment by PedanticAvenger** [comment]
alright, I finally got this up on github (been down with flu hard for more than a week).  You have a better handle on the data handling so I don't want to play around there.  I will work on this to add another "page" that can hold the graph.  Xdrip-Flashring is the repo

**Comment by PedanticAvenger** [comment]
You get a chance to look at that repo @Rytiggy?
At this point the "graph screen" code is in and I think I grabbed most of what you have for a settings page. At this point it just needs the data grab and internal messaging code. I also have to finish a design for a "ring like polling timer"bit that won't take long.

**Comment by Rytiggy** [comment]
Hey sorry, I've been super busy. I did get a chance to look over your code and it looks good. Did you want me to merge your code into this repo so that it can be centralized? I know its not an exact fork or my repo but i should be able to get  them working together

**Comment by PedanticAvenger** [comment]
Short answer yes, longer answer is I'd like to keep moving on some of the front face tweaks.  namely the "Time since last reading" component.  Whichever way is easier for me to get that integrated.  Can grab it later (probably tomorrow) or I can commit against whatever you grab.

**Comment by PedanticAvenger** [comment]
Alright, the graphical framework for the poll counter is there but it's throwing a few errors which are completely due to my ham handed hacking of JS.  Feel free to pull this.  I'll keep plugging at the poll stuff and if you don't tweaks will be there and if you do I can work against that.
(Unhandled TypeError: Argument at index 0 is not a Number
? at app/index.js:199,3
? at app/index.js:193,4
? at app/index.js:177,3
? at app/index.js:240,22)

**Comment by PedanticAvenger** [comment]
I'm also ok with granting publish on this repo if you think that could keep things cleaner as well.

**Comment by Rytiggy** [comment]
I can take a deeper look tomorrow but double check your data that is coming in is clean, no nulls, and double check that the keys in the Json blob are right for each point.

**Comment by PedanticAvenger** [comment]
Yeah, my issue was calling functions that are really just placeholders for your code and new code.  I've simply commented out the function calls and focused on the graphical framework effort.
